src/wallet/test/crypto_tests.cpp

```diff
@@ -23,14 +23,7 @@ bool OldSetKeyFromPassphrase(const SecureString& strKeyData, const std::vector<u
     if (nDerivationMethod == 0)
         i = EVP_BytesToKey(EVP_aes_256_cbc(), EVP_sha512(), &chSalt[0],
                           (unsigned char *)&strKeyData[0], strKeyData.size(), nRounds, chKey, chIV);
-
-    if (i != (int)WALLET_CRYPTO_KEY_SIZE)
-    {
-        memory_cleanse(chKey, sizeof(chKey));
```

Comment:
@jonasschnelli Yes, that is the correct fix if we want to keep the `memory_cleanse(…)`-calls. Do we? :-)  If so, I'll fix them!